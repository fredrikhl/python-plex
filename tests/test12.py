from plex import *
from utils import run_scanner

def test12():
    lex = Lexicon(
        [
            (Str("'") + Rep(AnyBut("'")) + Str("'"), TEXT),
            (AnyChar, IGNORE)
        ],
        debug = None,
        timings = None,
    )

    run_scanner(lex, "test12", trace = False)
