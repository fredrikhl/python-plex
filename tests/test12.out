test12.in   1   0 'Single-line text literal' "'Single-line text literal'"
test12.in   1  26 'Multi-
line text literal' "'Multi-\nline text literal'"
test12.in   5   0 None       ''
